packages/utils/src/types/ticket.ts

```diff
@@ -47,7 +47,7 @@ function serializeUnsigned({
 export class Ticket {
   constructor(
     readonly counterparty: Address,
-    readonly challenge: Hash,
+    readonly challenge: PublicKey,
```

Comment:
At the very least, needs an explanation _why_ this is a public key and not a hash.